test/parallel/test-assert.js

```diff
@@ -533,11 +551,18 @@ a.throws(makeBlock(thrower, TypeError), (err) => {
   const noop = () => {};
   assert.throws(
     () => { a.throws((noop)); },
-    common.expectsError({
-      code: 'ERR_ASSERTION',
-      message: /^Missing expected exception\.$/,
-      operator: 'throws'
-    }));
+    e => {
+      assert.strictEqual(e.code, 'ERR_ASSERTION');
+      assert(/^Missing expected exception\.$/.test(e.message));
+      assert.strictEqual(e.operator, 'throws');
```

Comment:
I wasn't confident to replace `common.expectsError()` with such custom assertions, but couldn't find a better way to reuse the existing test, as suggested.

Reply:
I would actually just keep this as is and just add a single line to one of the existing tests that checks for `assert.ok(!err.stack.includes('at throws')`.